### PR TITLE
Update base image for the jenkins slave to debian9

### LINKED ISSUE #237: Update base image for the jenkins slave
Currently jenkins slave is deployed using a debian8. We want to update it to debian9.

https://github.com/BeyondTheClouds/enos/blob/29ffa1e47ad550545989a0e39b2bfa34cbce6974/tests/functionnal/jenkins/common.py#L14****

### PR DESCRIPTION
**Pull request by msimonin**
This also update the vbox debian package to download

Fix #237

### CODE REVIEW

**Comment by msimonin** [comment]
I'd like to have this also in `stable/pike` before releasing the `enos` and the box.

File: tests/functionnal/jenkins/bootstrap.sh [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 set -x
 
-echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
-apt-get update && apt-get install -y -t jessie-backports openjdk-8-jre
```
Comment (dpertin) [inline tests/functionnal/jenkins/bootstrap.sh]: Don't we need:
`apt-get update && apt-get install -y openjdk-8-jre`
for Jenkins's slave?

Comment (msimonin) [reply tests/functionnal/jenkins/bootstrap.sh]: yes, we must. Let me fix that.

**Comment by msimonin** [comment]
@dpertin done. 
Seems ok, on the ci (at least the slave is correctly launched and vagrant/virtualbox packages installation is ok).

**Comment by dpertin** [comment]
Seems good to me :+1:

**Pull request merged**